### DISCUSSION TITLE
rclone support

### DISCUSSION THREAD

**Discussion by im7mortal** [ente-io/ente#616] [Enhancements]
**[rclone](https://github.com/rclone/rclone)** can considered as standard tool to copy between cloud storages considering number of supported services. 

I would like to be able to copy all data between ente and some others places. I know that you have very raliable storage _(Safely stored in [3 countries](https://ente.io/blog/replication-v3/), including an underground fallout shelter.)_ **the problem** is on user level. So far we are humans and it's possible to lost a key. It would be nice to keep back up. The rclone is good tool to manage this complexity.

**Comment by im7mortal** [comment]
From other side it's also would make very easy to import photos to ente from other clouds.

**Comment by vishnukvmd** [comment]
Hey, rclone is a great project, it's been on our radar.

For now we've started work on a [CLI](https://github.com/ente-io/cli) to simplify exports. Once shipped, it'll be easier to write a wrapper on top that integrates rclone.

Will consider moving this issue to CLI once we've launched v1.

**Reply by bingoxo** [reply]
now that v1 is out do you mind taking a look at it please ?

**Comment by im7mortal** [comment]
Do you mean you want to integrate rclone in CLI or CLI in rclone?

second is not really possible, I mean rclone can't call external CLI. I suppose rclone could have "ente" backend with libraries from CLI.

**Comment by hackardoX** [comment]
Hi @vishnukvmd, is there any update on this?
